rmf_robot_sim_gz_plugins/src/TeleportDispenser.cpp

```diff
@@ -92,7 +119,6 @@ class GZ_SIM_VISIBLE TeleportDispenserPlugin
 };
 
 TeleportDispenserPlugin::TeleportDispenserPlugin()
-: _dispenser_common(std::make_unique<TeleportDispenserCommon>())
 {
 }
```

Comment:
Now that the constructor does nothing we can safely remove it (or define it as `= default`).